tools/ci/init

```diff
@@ -101,21 +104,224 @@ def update_sdk(output_path: str, target_version: str):
     write_file(package_builder_path, package_builder_contents)
 
 
+def write_build_sh(output_path: str, framework_name: str):
+    contents = """
+#!/usr/bin/env bash
+set -e
+
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BUILD_DIR=$FRAMEWORK_DIR/build/distributions
+
+FRAMEWORK_NAME={{framework_name}}
+
+${FRAMEWORK_DIR}/gradlew -p ${FRAMEWORK_DIR} check distZip
+
+BOOTSTRAP_DIR=disable \\
+EXECUTOR_DIR=disable \\
+TEMPLATE_DOCUMENTATION_PATH="http://YOURNAMEHERE.COM/DOCS" \\
+TEMPLATE_ISSUES_PATH="http://YOURNAMEHERE.COM/SUPPORT" \\
+    $FRAMEWORK_DIR/tools/build_package.sh \\
+        ${FRAMEWORK_NAME} \\
+        $FRAMEWORK_DIR \\
+        -a "$BUILD_DIR/${FRAMEWORK_NAME}-scheduler.zip" \\
+        $@
+"""
+    write_file(os.path.join(output_path, "build.sh"),
+               contents.replace("{{framework_name}}", framework_name))
```

Comment:
I think we need a more unique replace target, this replaces `./bootstrap -template=false` with `./bootstrap -$framework_name=false` in marathon.json.mustache. Do you mind if push a fix for this @elezar?

Reply:
This only replaces the literal string `"{{framework_name}}"` in the local variable `contents` with `framework_name`.